README.md

```diff
@@ -1,93 +1,284 @@
+
+If the Steering Council membership exceeds its maximum, then all other Council business is suspended until this requirements is met. The requirement can be met by dropping Provisional members, removing members, or members voluntarily moving to Emeritus status.
+
+No vote can be held that would push the Steering Council above its maximum size if the vote resulted in adding either standard or provisional Steering Council members.
+
+### Provisional Members
+
+As mentioned above, Steering Council members will usually have a demonstrated record of prior service to the conda community.  This requirement works well for ensuring that Council members are invested in the community before they can become part of community governance.
+
+We also want to encourage participation by commercial and grant funded organizations that have committed to investing in the conda ecosystem. To encourage these organizations to commit resources to the conda ecosystem, **Provisional Steering Council Memberships** are available.
+
+Provisional memberships are open to organizations that have committed resources to the conda Organization, but do not yet have an established record of contribution *within* the conda Ecosystem. Provisional memberships encourage investment, and give investing organizations an immediate voice in community decisions.
+
+Provisional memberships are open to organizations that
+
+* Are funding at least one FTE contributing to the conda ecosystem
+* Have not been involved in the conda ecosystem previously (and thus have no record of contributing to the ecosystem)
+
+These organizations can request 1 provisional membership on the Steering Council. The evaluation criteria for evaluating this request is based on the organization’s and the nominated person’s commitment to open source principles *as demonstrated in other projects.*
+
+Provisional memberships are expected to be temporary, lasting just long enough for contributors from that organization to demonstrate their values and earn representation as individual contributors.
+
+Provisional memberships can also be revoked at any time by a simple majority vote of the Steering Council, if and when the Council determines that the organization or the individual is no longer acting in the larger interests of the conda community.
+
+Finally, **no more than 1/3 of the Steering Council membership can be Provisional Members.** At all times, at least 2/3 of the Steering Council membership hold a seat because of demonstrated contributions to the conda ecosystem. If this threshold is violated then all other Council business is suspended until this requirements is met. The requirement can be met by dropping Provisional members, or by adding contributing members.
```

Comment:
See above discussion of "all other Council business"

I am making this comment solely in my personal capacity and am not conveying any rights to any intellectual property of any third parties.

Reply:
Got it.  Will use whatever text we settle on above.

Reply:
This has been addressed.